README.md

```diff
@@ -27,3 +27,16 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+
+Answer: 
+Invariant Definition: 
+Looking back at the slides. The invariant or proving an invariant is we have to prove the invariant exists before the loop begins. Then at the end of the iteration prove that it holds the value from the previous iteration. 
+
+Within my code, the invariant would be fib(a-1). The reason is that the recursive sequence will go up to (a - 1). At each start of the recursive call, the arr is filled up with the Fibonacci sequence up to (a - 1). After that the next step is to calculate Fib(a) by summing up the two values at (arr[arr.length - 1] + arr[arr.length - 2]). With this invariant, it helps ensure that the Fibonacci sequence is done correctly step by step.
```

Comment:
Not sure what's going on here. I've told you repeatedly that a statement like you're making is *not* a valid invariant, yet you keep requesting reviews for the same invariant. Please ask the TA or me is this is unclear, but I won't review this again until you change the invariant.